edge-modules/SimulatedTemperatureSensor/docker/windows/arm32v7/base/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Base image for arm32v7 dotnet-runtime container
 FROM mcr.microsoft.com/windows/nanoserver:1809-arm32v7
  
-ENV DOTNET_VERSION 2.1.13
+ENV DOTNET_VERSION 3.1.2
 ENV DOTNET_DOWNLOAD_URL https://dotnetcli.blob.core.windows.net/dotnet/Runtime/$DOTNET_VERSION/dotnet-runtime-${DOTNET_VERSION}-win-arm.zip
  
 RUN curl -SL %DOTNET_DOWNLOAD_URL% --output dotnet.zip \
```

Comment:
We have removed winarm32 from the build/test pipelines, but we still have the Dockerfiles in the codebase. I would like to remove these all of these if possible to remain consistent with our supported tiers. IMO it is dangerous to have something here remaining untested that can and will be broken in the future.

If we do decide to keep these base images, we will have to also add the aspnet framework dependency.

Reply:
I spoke with Varun and we decided to leave the dockerfiles but not update them.